### ISSUE TITLE
Search method

### ISSUE THREAD

**Issue by edapm** [BibleJS/bible-english#10]
Issue #3 requested this. I think maybe trying to download the JSON locally as in [Bible-RO](https://github.com/biblejs/bible-romanian) is a good idea?

**Comment by IonicaBizau** [comment]
Yes, I think that can work, but we have to make sure the copyright rules allow us to do that.
